pkg/etcdutil/init.go

```diff
@@ -37,6 +38,7 @@ func (c *EtcdConnectionConfig) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&c.Username, "etcd-username", c.Username, "etcd server username, if one is required")
 	fs.StringVar(&c.Password, "etcd-password", c.Password, "etcd server password, if one is required")
 	fs.DurationVar(&c.ConnectionTimeout.Duration, "etcd-connection-timeout", c.ConnectionTimeout.Duration, "etcd client connection timeout")
+	fs.DurationVar(&c.SnapshotTimeout.Duration, "etcd-snapshot-timeout", c.SnapshotTimeout.Duration, "context timeout for taking etcd snapshots")
```

Comment:
`context timeout for taking etcd snapshots` -> `timeout duration for taking etcd snapshots`